src/windowmanagement/HelpMenu.tsx

```diff
@@ -1,26 +1,97 @@
 
 export function HelpMenu() {
+  const luaApi = useOpenSpaceApi();
+  const [opened, { open, close }] = useDisclosure(false);
+  const [isConsoleVisible, setIsConsoleVisible] =
+    useGetBoolPropertyValue('LuaConsole.IsVisible');
+  const [portProperty] = useGetIntPropertyValue('Modules.WebGui.Port');
+  const [addressProperty] = useGetStringPropertyValue('Modules.WebGui.Address');
+
+  function toggleLuaConsole() {
+    setIsConsoleVisible(!isConsoleVisible);
+  }
+
+  function openGuiInBrowser() {
+    const port = portProperty ?? 4670; // TODO: Change this to correct port when hosted
+    const address = addressProperty ?? 'localhost';
+
+    const link = `http://${address}:${port}`;
+    window.open(link, '_blank');
+  }
+
+  function toggleShutdown() {
+    luaApi?.toggleShutdown();
+  }
+
   return (
-    <Menu position={'bottom-start'} offset={4} withArrow arrowPosition={'center'}>
-      <Menu.Target>
-        <Button size={'xs'} color={'gray'}>
-          Help
-        </Button>
-      </Menu.Target>
-      <Menu.Dropdown>
-        <Menu.Item>Open Web Tutorials</Menu.Item>
-        <Menu.Item>Open Getting Started Tour</Menu.Item>
-        <Menu.Divider />
-        <Menu.Item>Send Feedback</Menu.Item>
-        <Menu.Divider />
-        <Menu.Item>Toggle Console</Menu.Item>
-        <Menu.Item>Open GUI in Browser</Menu.Item>
-        <Menu.Divider />
-        <Menu.Item>Quit OpenSpace</Menu.Item>
-        <Menu.Divider />
-        <Menu.Item>About</Menu.Item>
-      </Menu.Dropdown>
-    </Menu>
+    <>
+      <About opened={opened} close={close} />
+
+      <Menu position={'bottom-start'} offset={4} withArrow arrowPosition={'center'}>
+        <Menu.Target>
+          <Button size={'xs'} color={'gray'}>
+            Help
+          </Button>
+        </Menu.Target>
+        <Menu.Dropdown>
+          <Menu.Item
+            component={'a'}
+            href={
+              'https://www.youtube.com/playlist?list=PLzXWit_1TXsu23I8Nh2WZhN9msWG_ZbnV'
+            }
+            target={'_blank'}
+            leftSection={<BookIcon />}
+          >
+            Open Web Tutorials
+          </Menu.Item>
+          <Menu.Item leftSection={<RouteIcon style={{ transform: 'scale(-1)' }} />}>
+            Open Getting Started Tour
+          </Menu.Item>
+          <Menu.Divider />
+          <Menu.Item
+            component={'a'}
+            href={'http://data.openspaceproject.com/feedback'}
+            target={'_blank'}
+            leftSection={<FeedbackIcon />}
+          >
+            Send Feedback
+          </Menu.Item>
+          <Menu.Divider />
+          <Menu.Item onClick={toggleLuaConsole} leftSection={<ConsoleIcon />}>
+            Toggle Console
+          </Menu.Item>
+          <Menu.Item onClick={openGuiInBrowser} leftSection={<OpenInBrowserIcon />}>
+            Open GUI in Browser
+          </Menu.Item>
+          <Menu.Divider />
+          <Menu.Item onClick={toggleShutdown} leftSection={<ExitAppIcon />}>
```

Comment:
Clicking this menu item should open a modal that says something like "Do you really want to exit the application? Confirm / Cancel"